.github/workflows/pr-realm-web.yml

```diff
@@ -32,27 +35,22 @@ jobs:
         with:
           node-version: 20
           registry-url: https://registry.npmjs.org/
-      # Install the root package (--ignore-scripts to avoid downloading or building the native module)
-      - run: npm ci --ignore-scripts
-      # Bootstrap lerna sub-packages
-      - run: npx lerna bootstrap --scope realm-web-integration-tests --include-dependencies
+      # Install the root package
```

Comment:
I wonder if this entire workflow could be replaced by the `package-unit-tests`?

Reply:
Interesting thought .. it probably could. But there's something in the naming of that workflow that goes against running "integration tests" from it. For now, I think it'll be best to keep separate if we choose to merge `realm-web` into `realm` over time 🤞

Reply:
I guess my idea was any of our packages could be run by `package-unit-tests` (perhaps the naming could be improve).  I originally attempted to get `realm` and `realm-web` in there, but `realm` had too much special needs and `realm-web` needed the work you are merging here.